tests/test_pickling_trampoline.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2021 The Pybind Development Team.
+// All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+#include "pybind11_tests.h"
+
+#include <memory>
+#include <stdexcept>
+#include <utility>
+
+namespace {
+
+struct SimpleBase {
+    int num               = 0;
```

Comment:
nit Whitespace alignment is inconsistent with other files.

Can you remove visual indents for now? e.g. `sed -E 's#(\w) +#\1 #g'`?